### DISCUSSION TITLE
Adapter le message du mini-event des récompenses de ligue du dimanche en fonction du classement du joueur dans la saison précédente

### DISCUSSION THREAD

**Discussion by Nargacaura** [Crownicles/Crownicles#1870] [Nouvelles idées]
Ça me parait un peu bête de tomber sur le mini-event un dimanche qui nous annonce:
> "Les récompenses de ligues sont disponibles ! Récupérez votre récompense avant la fin de la semaine grâce à la commande /bonusligue !"

et que lorsqu'un joueur utilise cette commande, il a des chances de se faire troller car il n'a pas été classé durant la saison précédente. Pourquoi ne pas l'adapter en fonction du classement dans la saison précédente? Ce que je veux dire par là, c'est en gros faire en sorte à ce que le message ci-dessus ne s'affiche que s'il était classé, sinon mettre un message du style (le message sera à modifier, of course):
> "Les récompenses de ligues sont disponibles ! Malheureusement, vous n'avez pas réussi à vous classer durant la saison précédente, et donc ne pouvez pas récupérer de récompense..."

Bien sûr, ça ne s'applique que pour le message du dimanche dans le mini-event. Celui des autres jours peut rester comme il est.

**Comment by KirIcare** [comment]
Ça me paraît cool
Et dans le cas où t'as déjà récupéré tes récompenses ? Un truc du genre "revenez la semaine prochaine" ou on le laisse comme ça ?

**Reply by Nargacaura** [reply]
Ça pourrait aussi être une solution envisageable de modifier en conséquence le message en quelque chose du style:
> "Les récompenses de ligues sont disponibles ! Cependant, vous avez déjà récupéré votre récompense, revenez lorsque la saison se terminera samedi prochain !"

Bon, après, faudra voir pour changer "samedi prochain" en un timestamp donnant le countdown avant la fin de la saison. Cependant, faudra aussi voir en fonction de la complexité de ce changement.
